src/main/java/PageObjects/MyTicketPage.java

```diff
@@ -124,4 +152,30 @@ public boolean verifyTicketWasDeleted(String actualMsg, String expectedMsg) {
             flag = true;
         return flag;
     }
+
+    public boolean isTitleFilterForm(){
+        boolean flag = false;
+        if (this.getTitleFilterElement().getText().equals(Constant.TITLE_FILTER_FORM)){
+            flag = true;
+        }
+        return flag;
+    }
+
+    public static String getAmountTicket(){
+        By amountTicket = By.xpath("//table[@class='MyTable']//td[count(//table[@class='MyTable']//th[text()= 'Status']/preceding-sibling::th)+1][text()='New']");
+        return String.valueOf(Constant.driver.findElements(amountTicket).size());
+    }
+
+    public String getNote() {
+        return Constant.driver.findElement(lblNote).getText();
+    }
+
+    public boolean checkAmountTicket(){
+        String text = " book "+ getAmountTicket() +" tickets";
```

Comment:
When you book 1 ticket only, the text should be _book 1 ticket_ 
Review your checkpoint again